### PR TITLE
Postcard - Data views updates

### PR DESCRIPTION
**Pull request by osimarkrogers**

### CODE REVIEW

**Comment by rmuolic** [comment]
@jamesbperry Hi Brandon, per James Golla's update on the support for non-datetime index types feature being back in for Postcard, I re-added back in the updated content for quick-start-get-data-view-data.md and quick-start-define-a-data-view.md for from your [PR 611](https://github.com/osisoft/OCS-Docs/pull/611/files).

**Review by jamesbperry — APPROVED** [review]

File: content/add-organize-data/organize-data/data-views/data-views.md [hunk 1]
```diff
@@ -8,32 +8,46 @@ Data views are subsets of data from one or more streams. You create a data view
...
 
+1. Click the **Items per page** dropdown list to select the number of items to be displayed per page in the data view.
 
+1. Click the **Interpolated/Stored** dropdown list and select **Interpolated** or **Stored** for the data view.
+
+1. Click the date and time display to set the **Start Index** and **End Index** to enter the time period of the data to include in the data view. For interpolated data views, enter a `Time Interval`.
+
+1. Click **Apply** to accept the index configuration changes.
+
+1. Click the **View in API Console** button to display basic parameters of the data view in the API console.
```
Comment (osisoft-mbishop) [inline content/add-organize-data/organize-data/data-views/data-views.md:47]: Again, I think you can omit this step. Viewing the API console doesn't affect user outcome when creating a data view. Information about the API console in relation to data views might be better as a stand-alone task.

```suggestion
```

File: content/api-reference/data-views/get-data-view-data/quick-start-get-data-view-data.md [hunk 2]
```diff
@@ -9,12 +9,12 @@ You retrieve data view data with the [Data API](xref:DataViewsDataAPI). This top
...
+All data in a data view is associated with an index value, for example, a timestamp. If there are multiple groups in the data view, the index and grouping field values together form a unique identifier for each data record.
 
 #### Index type
-The supported index type is `DateTime`, as noted in [Define a Data View](xref:DataViewsQuickStartDefine). Date times should use ISO 8601 format and can be in either UTC or local time. Data retrieved using the [Data API](xref:DataViewsDataAPI) will be returned in UTC.
+Though `DateTime` is the most common index type for stream data, data views can target data of other index types, as noted in [Define a Data View](xref:DataViewsQuickStartDefine).
 
-Interval timespan values should be formatted using the standard timespan format, ie., d.hh:mm:ss.fffffff.
+For `DateTime` indexed data views, index values should use ISO 8601 format and can be in either UTC or local time with an offset specified. Data retrieved using the [Data API](xref:DataViewsDataAPI) will be returned in UTC. Interval timespan values should be formatted using the standard timespan format, ie., d.hh:mm:ss.fffffff.
```
Comment (osisoft-mbishop) [inline content/api-reference/data-views/get-data-view-data/quick-start-get-data-view-data.md:17]: Rewrite this in present tense. Make it active voice as well. Suggestion:

```suggestion
For `DateTime` indexed data views, use the ISO 8601 format  as the index value. Format interval timespan values using the standard timespan format. For example, d.hh:mm:ss.fffffff. Use either UTC or local time with an offset specified. Data retrieved using the [Data API](xref:DataViewsDataAPI) returns in UTC.
```

**Review by osisoft-mbishop — CHANGES_REQUESTED** [review]
From a doc qa perspective, I have a few style guide suggestions, but otherwise ok.

From a peer review perspective, not so much. I know you aren't the original author, but I had a hard time following the step-by-step:

* The doc references labels that don't appear in the UI, which can be confusing.
* Many of the steps lack context for what certain options do. For example, I'm lost on what "Grouping Fields" and "Identifying Fields" options are for.
* Additionally, there are lots of references to different panes and similarly named UI elements, which are easy to get lost in. 

I also think this procedure might benefit being broken into a "mega task": 1. Create a data view, 2. Add queries to data view, 3. Configure queries, etc.

I also think adding some screen caps would be helpful. I know they're difficult to manage, but in this situation, with a combination of unlabled panes or similarly labled fields, they would be helpful.

Sorry for the dings, as I know you aren't the OG author here. I know much of this feedback likely can't be addressed this sprint, but we should think about some rewrites in the future.  Let me know if you want to talk about any of them in person.

File: content/add-organize-data/organize-data/data-views/data-views.md [hunk 3]
```diff
@@ -8,32 +8,46 @@ Data views are subsets of data from one or more streams. You create a data view
...
-10. Select **Save Default with Data View** to save your settings with this data view.
-11. In the `Field Management` pane, click **Add**. Then in the `Add Data Fields` window, select the data fields from the query results to include in the data view and click **Add**.
-12. In the `Field Management` pane, use the **Up** and **Down** Arrow buttons to move selected data fields up or down in the display, and click **Apply**.
-13. Click **Save**.
 
-## Related links
+1. Select any stream in the query results to display the individual data fields of the stream in the `Data Item Details` pane.
 
-To define a data view using the OCS API, see [Define a data view](xref:DataViewsQuickStartDefine).
+   **Note**: You can click the **Add Query** button to create as addtional queries.
```
Comment (rmuolic) [inline content/add-organize-data/organize-data/data-views/data-views.md:29]:
```suggestion
1. Select the **Add** button to add additional queries.
```

Comment (rmuolic) [inline content/add-organize-data/organize-data/data-views/data-views.md:29]:
```suggestion
1. Select the **Add** button to add additional data fields to the query.
```

**Pull request closed without merging**